lobster_simulator/Simulator.py

```diff
@@ -8,39 +8,53 @@
 from pkg_resources import resource_stream
 
 from lobster_simulator.robot.Lobster import Lobster
-from lobster_simulator.simulation_time import SimulationTime, microseconds_to_seconds, seconds_to_microseconds
+from lobster_simulator.simulation_time import SimulationTime
+from enum import Enum
+
+
+class Models(Enum):
+    SCOUT_ALPHA = 1
```

Comment:
I think you can remove the numbers from this Enum. 
```
```suggestion
    SCOUT_ALPHA
    PTV
```

Reply:
I tried this, but that gives an error, but I did notice that you can use `= auto()` if the value doesn't matter.